### PR TITLE
Updating the error handling part

### PR DESCRIPTION
**Pull request by prabhuyadav**
updated the error handling part (initial update, can be revised later) and  changed variable declarations from let to const. Haven't tested by connecting to our remote mongo db yet, facing some problems with it from my local.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -13,23 +12,24 @@ app.use(bodyParser.json());
...
 const options = {useNewUrlParser: true, useUnifiedTopology: true}
 const mongo = mongoose.connect(dbPath, options);
 mongo.then(() => {
     console.log('connected');
-}, error => {
+}).catch(error => {
     console.log(error, 'error');
-})
\ No newline at end of file
+    process.exit(1);
```
Comment (phoenixnitin) [inline index.js:34]: Why exit?

**Comment by phoenixnitin** [comment]
Why merging to mounika branch

**Comment by phoenixnitin** [comment]
You can merge to main branch

**Comment by prabhuyadav** [comment]
> You can merge to main branch

ok. i just thought since it is not tested that is why didn't want to merge to main branch.

**Review by phoenixnitin — APPROVED** [review]

**Pull request merged**